tests/test_uploads_from_komodo.py

```diff
@@ -180,32 +176,22 @@ def test_case_tables(explorer: Explorer):
     cases = _get_suitable_cases(explorer)
     perfect_cases = 0
     for case in cases:
+        tbls = case.tables
+        assert len(tbls.uuids) == len(tbls)
+        assert sum(len(tbls.filter(name=n)) for n in tbls.names) == len(tbls)
+
         realizations = len(case.realizations)
-        ens_count = 0
-        real_count = 0
-        tagname_count = 0
-        # SIM2SUMO uploads:
-        drogon_rft_count = 0
-        drogon_satfunc_count = 0
-        drogon_summary_count = 0
-
-        for tbl in case.tables:
-            assert tbl.uuid
-            assert tbl.name
-            if tbl.metadata.get("fmu").get("aggregation") is not None:
-                continue
-            if tbl.ensemble is not None:
-                ens_count += 1
-            if tbl.realization is not None:
-                real_count += 1
-            if tbl.tagname is not None:
-                tagname_count += 1
-            if tbl.name == "DROGON" and tbl.tagname == "rft":
-                drogon_rft_count += 1
-            if tbl.name == "DROGON" and tbl.tagname == "satfunc":
-                drogon_satfunc_count += 1
-            if tbl.name == "DROGON" and tbl.tagname == "summary":
-                drogon_summary_count += 1
+        ens_count = len(tbls.filter(ensemble=True))
+        real_count = len(tbls.filter(realization=True))
+        tagname_count = len(tbls.filter(tagname=tbls.tagnames))
```

Comment:
Can we just use `len(tbls.tagnames)` here?

Reply:
Not really, tbls.tagnames returns only unique tagnames.